### ISSUE TITLE
Generate only alive patients

### ISSUE THREAD

**Issue by flowrider3000** [synthetichealth/synthea#624]
Hi,
is it possible to generate only alive patients with `synthea`?
Best

**Comment by jawalonoski** [comment]
I have to admit that it is somewhat strange, but the answer is **no**.

We would take a pull request to add this as a feature, if you are interested in writing that code. Otherwise, we'll add it eventually.

There is a configuration setting to generate only dead patients in `synthea.properties`:

``` properties
# Set to true if you want every patient to be dead.
generate.only_dead_patients = false
```

The explanation on why we don't have this option today, is historical. Originally, we used Synthea to generate an entire population for a town, for example. If someone says "_the population of Whoville is 152 people_" then you would set the population flag to be `-p 152`. Some of the patients generated would die along the way, but you want to ignore them, because when someone tells you the population of a town, they are always talking about the number of living people. Rather than just discard the patients that randomly died, we decided to save them. Presumably Electronic Health Record (EHR) systems will have historical records from dead patients in their systems. That was the thinking, anyway.

**Comment by flowrider3000** [comment]
I already thought something like that ;)
However, would it be ok for you to create a pull request to add this feature?

**Comment by jawalonoski** [comment]
Fixed with #655
